platform_versioned_docs/version-24.1/supported_software/agent/agent.mdx

```diff
@@ -22,6 +22,13 @@ Tower Agent is distributed as a single executable file to simply download and ex
 
 2. (Optional) Move it to a folder that's in your $PATH.
 
+### Environment variables
+
+The following environment variables are available:
+
+- `TW_AGENT_WORKDIR`: Sets the work directory for the agent. By default, this directory resolves to the `${HOME}/work` path, where `HOME` is the home directory of the user that the agent runs as. The work directory can also be overridden by specifying the `--work-dir` argument.
+- `TW_AGENT_USER`: Sets the user name that the agent identifies as running as. By default, this is the Linux user name that the agent runs as and is the value of the `USER` environment variable.
+
```

Comment:
These are not environment variables, these are variables that you can use when you create the compute environment to define a variable working directory.

Given that a CE that uses Agent can be used by multiple users, they may want that each user uses a different directory as working directory.

**Examples:**
1. They want all the users to use the same directory as the working directory; they just hardcode the full path when creating the CE. On many HPCs, this is problematic because the permissions do not allow multiple users to use the same folder.
![Screenshot from 2024-10-02 06-08-23](https://github.com/user-attachments/assets/855d4269-133c-4bf6-8843-204c50f8b1e2)

2. They want to decide where the users are going to store the working files, but they want a different folder for each user. Then they can use `TW_AGENT_USER` variable:
![Screenshot from 2024-10-02 06-02-36](https://github.com/user-attachments/assets/3d11ad1d-7c6f-48cd-bac4-206092931a4a)

3. They want the user to fully decide where to store the working files. Then they just use `TW_AGENT_WORKDIR` that is a full path to somewhere that the user decided when launching the agent. 
![Screenshot from 2024-10-02 06-01-33](https://github.com/user-attachments/assets/5482f361-5d30-4b40-b3e1-c1dff47e01ed)

**How the Agent launcher can decide the value of these variables?**

- When you launch the Agent, the workdir is `$HOME/work` by default unless you specify a different one using `--work-dir`. 
- The username is the Unix username that launches the Agent. On a few systems, the Agent fails to automatically discover it and falls back to `USER` environment variable value.